### ISSUE TITLE
calculate_attenuation_coefficients with NOPMRT option raising error

### ISSUE THREAD

**Issue by katerinachytra** [UCL/STIR#1515]
It seems that a  condition with NOPMRT option is needed to --argc before arg count check to avoid print_usage_and_exit.

**Comment by KrisThielemans** [comment]
Indeed. At present. the `--NOPRMT` option is no longer functional (and `--PMRT` is the default),

I'm a bit reluctant to fix this, as the old `ForwardProjectorByBinUsingRayTracing` does not support many systems. I'd rather remove this option from the usage message.

Any reason why you need the `ForwardProjectorByBinUsingRayTracing`?

(sorry that I answer this so late)
